cms/jinja2/templates/datavis/_macro.njk

```diff
@@ -33,5 +31,15 @@
                 {% endif %}
             </ol>
         {% endif %}
+
+        <!-- script tags to pass json data to the javascript component -->
```

Comment:
Nit: Whilst I like the comment, I'd suggest it be a jinja2 comment rather than an HTML comment, to keep it out the final page

```suggestion
        {# script tags to pass json data to the javascript component #}
```

Reply:
agreed! Updated.